templates/projects/directory.html

```diff
@@ -35,292 +35,278 @@ <h1>Open source projects directory</h1>
               <th scope="col">Description</th>
               <th scope="col">Module/project name</th>
               <th scope="col">Contact</th>
-              <th scope="col">Contact email</th>
             </tr>
           </thead>
           <tbody>
             <tr>
-              <th scope="row">Candid authentication service</th>
-              <td>candid</td>
-              <td>Francesco Banconi</td>
-              <td>francesco.banconi<span class="sign sign--first">{ AT }</span>canonical<span class="sign sign--second">{ DOT }</span>com</td>
+            <th scope='row'>Candid authentication service</th>
```

Comment:
I think your formatter should be reconfigured. We use double quotes in HTML.

Reply:
Also, the indentation looks weird, `th` needs to be indented from `tr`, basically how it was before

Reply:
Oh this mark up is generated by the spreadsheet but I can format it.